gateway/tests/api/test_v1_program.py

```diff
@@ -106,14 +106,6 @@ def test_run(self):
                 "arguments": arguments,
                 "dependencies": "[]",
                 "env_vars": env_vars,
-                "config": json.dumps(
```

Comment:
I want to propose modify the test not remove. It should be something as simple as send the `json`, should not?

Reply:
This test should put a `dict` here but the test client `APITestCase.client.post` doesn't support **multipart uploads with nested data**.  So I took it out for now.

Reply:
Sorry @akihikokuroda , I didn't have the time to see your comment until now. I tested it locally and for me is working the `json.dumps`, is it returning you an error?

Reply:
Based on https://github.com/Qiskit-Extensions/quantum-serverless/pull/1297#discussion_r1589287485.  I take put this "test_run" test :-)

Reply:
Hahaha fair point. Let's do that then 👍